spec/supabase_js_v2.yml

```diff
@@ -3674,23 +2270,20 @@ functions:
               "statusText": "OK"
             }
             ```
-
-          </TabPanel>
-          </Tabs>
         hideCodeBlock: true
         isSpotlight: true
-  - id: ilike
-    title: ilike()
-    $ref: '@supabase/postgrest-js.PostgrestFilterBuilder.ilike'
+  - id: is
```

Comment:
possibly a mistake here?
might just be GitHub UI showing it weird.

did we lose `ilike` function ?

Reply:
I think it might be Github showing it weird as I can see `ilike` on line 2234:

<img width="730" alt="Screenshot 2022-12-15 at 08 47 17" src="https://user-images.githubusercontent.com/22655069/207814300-38d36d9c-0883-45b8-be48-9622eaa2c1a3.png">

Reply:
👍